src/components/data-table/table-row-actions.tsx

```diff
@@ -25,6 +26,7 @@ export default function TableRowActions(props: TableRowActionsProps) {
     };
 
     const handleDelete = () => {
+        dispatch(removeInvalidColineOnRemove(stationId));
```

Comment:
Can this action be dispatched in remove-station action? Also please handle the case when station is not removable.

Reply:
I remove most logic in the checks so there should be no need for tests 🤣